src/plugins/navigation/public/top_nav_menu/top_nav_menu_item.tsx

```diff
@@ -49,7 +49,7 @@ export function TopNavMenuItem(props: TopNavMenuData) {
   };
 
   const btn = props.emphasize ? (
-    <EuiButton {...commonButtonProps} size="s" fill style={{ fontSize: 'smaller' }}>
+    <EuiButton {...commonButtonProps} size="s" fill>
```

Comment:
Changing this to size "S" increases the height of the nav row and also creates a weird visual effect when switching edit mode on and off. Is that OK?

Reply:
With Devon's original PR (adding this button) we accepted the height change between view and edit modes. This particular change here only affects the font-size and has no impact on the overall button height. 

This situation will be addressed when we move to the new header design where the menus are anticipated to move up into the header area - the plan as of now is for a double header, like GitHub, and will better accommodate buttons in the menu layout.

Reply:
@lizozom given that explanation, can you approve this PR? Thanks!